src/app/query/stacks/bns/bns.hooks.ts

```diff
@@ -10,6 +10,13 @@ export function useCurrentAccountNames() {
   return useMemo(() => namesResponse.data?.names ?? [], [namesResponse.data?.names]);
 }
 
+export function useCurrentAccountNamesQuery() {
+  const principal = useCurrentAccountStxAddressState();
+  const namesQuery = useGetBnsNamesOwnedByAddress(principal);
+  if (principal === '') logger.error('No principal defined');
```

Comment:
As this is an exceptional case that shouldn't happen, this is okay, but we should be careful logging with the main body of a hook/component, as it'll trigger on every rerender (if principal _is_ `''` then it'll log many many times)

Reply:
Alright, I used the `useCurrentAccountNames()` hook just above this one as a template when creating this hook, which also uses the logger. I'm still getting the hang of what is and isn't needed. 

Seeing that it should be an exceptional case, should we use these loggers for the time being?